src/Foundation/NSHost.cs

```diff
@@ -155,6 +168,11 @@ IEnumerator IEnumerable.GetEnumerator ()
 		{
 			return GetEnumerator ();
 		}
+
+		bool IEquatable<NSHost>.Equals (NSHost? other)
+		{
+			return Equals ((object?) other);
```

Comment:
Using as is the same:
```suggestion
			return Equals (other as object);
```

Reply:
It looks a bit weird though, someone might think "that as check is always going to succeed, so let's just remove it".